measurement/plugins/webpage_download/measurements.py

```diff
@@ -0,0 +1,172 @@
+import time
+import requests
+from six.moves.urllib.parse import urlparse
+from bs4 import BeautifulSoup
```

Comment:
I get an import error even after `poetry install`ing.

Don't add dependencies manually. Add them via `poetry add` to ensure they're added to `pyproject.toml` and the versions are locked in `poetry.lock`.

```
ImportError while importing test module '/home/james/dev/honestybox/honestybox-measurement/measurement/plugins/webpage_download/tests/test_measurements.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
measurement/plugins/webpage_download/tests/test_measurements.py:10: in <module>
    from measurement.plugins.webpage_download.measurements import WebpageMeasurement
measurement/plugins/webpage_download/measurements.py:4: in <module>
    from bs4 import BeautifulSoup
E   ModuleNotFoundError: No module named 'bs4'
```